tests/test_restore_names.py

```diff
@@ -1,3 +1,90 @@
 from app.restore_names import restore_names
 
-# write your tests here
+
+def test_name_none():
```

Comment:
```suggestion
def test_first_name_equals_none():
```
Let's use more informative names for test